src/Swarm/Game/ScenarioInfo.hs

```diff
@@ -177,7 +177,7 @@ updateScenarioInfoOnQuit z ticks completed (ScenarioInfo p s bTime bTicks) = cas
 -- | A scenario item is either a specific scenario, or a collection of
 --   scenarios (*e.g.* the scenarios contained in a subdirectory).
 data ScenarioItem = SISingle ScenarioInfoPair | SICollection Text ScenarioCollection
-  deriving (Eq, Show)
+  deriving (Show)
```

Comment:
Removed `Eq` instance due to adding `PrerequisiteConfig` to `Objective`, which doesn't support `Eq`.

Reply:
@kostmo what do you mean? `PrerequisiteConfig` has `Eq`.

Reply:
You're right, not sure what I was thinking.  I've restored them.